### DISCUSSION TITLE
FlySky AFHDS 2A  -  OpenTx (2.3.x) RSSI alarm threshold settings

### DISCUSSION THREAD

**Discussion by franz4949** [pascallanger/DIY-Multiprotocol-TX-Module#714] [Q&A]
In the protocol details it is recommended for iA6B / iA10B receivers to set RSSI treshold low at 70 and treshold critical at 50. Based on my own tests with iA6B I also would set the RSSI critical value at 50, but why do you select to set the treshold for low at 70? That is 20 dB above the criticak value, an unusual high difference? Is there an extraordinary reason for it? Or wouldn't it be more reasonable to set the RSSI treshold low at 60 maybe?

**Comment by pascallanger** [comment]
This is based on users feedback that have compared the original radio thresholds with multi. They came up with these numbers that I haven't challenged l. It is given as a start for config. After you can adjust the values like you want.

**Reply by franz4949** [reply]
Okay, understood, so I will set my treshold low at 60. According my experience that's appropriate for safe flying and does avoid unnecessary and annoying too early warnings, Thanks for your reply and your phantastic work.
